src/index.ts

```diff
@@ -214,26 +207,10 @@ class Monika extends Command {
       startPrometheusMetricsServer(flags.prometheus)
     }
 
-    const isOpenConfigGenPage = this.isOpenConfigGeneratorPage(flags)
-    if (isOpenConfigGenPage) {
-      log.info(
-        'Opening Monika Configuration Generator in your default browser...'
-      )
-      await open('https://hyperjumptech.github.io/monika-config-generator/')
-      return
-    }
-
     try {
-      if (isUrl(flags.config)) {
-        await setupConfigFromUrl(flags.config, flags['config-interval'])
-      } else {
-        const watchConfigFile = !(
-          process.env.CI ||
-          process.env.NODE_ENV === 'test' ||
-          flags.repeat
-        )
-
-        await setupConfigFromFile(flags, watchConfigFile)
+      await setupConfig(flags)
+      if (flags['create-config']) {
+        return
```

Comment:
we don't need to put this here. check my comment above. we can instead call `createConfig` as early as possible when the `run` function starts.

```javascript
  async run() {
    const { flags } = this.parse(Monika)

    if (flags['create-config']) {
      await createConfig()
      return
    }
```

Reply:
moved